Formula/grpc-php.rb

```diff
@@ -4,8 +4,8 @@
 class GrpcPhp < Formula
   desc "gRPC PHP extension"
   homepage "http://www.grpc.io/"
-  head "https://github.com/grpc/grpc.git", :branch => "release-0_9"
-  version "0.9.1"
+  head "https://github.com/grpc/grpc.git", :branch => "release-0_10"
```

Comment:
is there a reason why we don't pick PHP from the 0_10_0 tag (as opposed to  0_10 branch)?